packages/adapters/eventstore-adapters/README.md

```diff
@@ -36,86 +12,7 @@ Available adapters:
   Used to store events in a PostgreSQL.
 - [resolve-eventstore-postgresql-serverless](./resolve-eventstore-postgresql-serverless)  
    Used to store events in a PostgreSQL Serverless.  
```

Comment:
MySQL, PostgreSQL and PostgreSQL Serverless are all database names, right? If so, is it technically correct if we say:

Line 34/10:
`Used to store events in a MySQL database.`

Line 36/12:
`Used to store events in a PostgreSQL database.`

Line 38/14:
`Used to store events in a PostgreSQL Serverless database.`

Reply:
fixed